bin/release

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+source ./venv/bin/activate
+
+set -ux
+rm -r ./dist/
+git fetch yelp
```

Comment:
That's not different from the LHS, and I don't see a way to do without assumptions.